modules/test/playwright/fixtures/login.fixture.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect, test} from '@playwright/test';
+
+import {liferayConfig} from '../liferay.config';
+import {createTempFile, readTempFile} from '../utils/util';
+
+export interface Login {
+	password: string;
+	sessionId: string;
+	user: string;
+}
+
+let loggedIn = false;
+
+const loginTest = test.extend<{
+	_login: Login;
+}>({
+	_login: async ({page}, use) => {
+		const user = liferayConfig.user.login;
+		const password = liferayConfig.user.password;
+
+		if (!loggedIn) {
+			const storageStatePath = createTempFile('storageState.json');
+
+			await page.goto('/');
+
+			await page.getByRole('button', {name: 'Sign In'}).click();
+
+			await page.getByLabel('Email Address').fill(user);
+			await page.getByLabel('Password').fill(password);
+			await page.getByLabel('Remember Me').check();
+
+			await page
+				.getByLabel('Sign In- Loading')
+				.getByRole('button', {name: 'Sign In'})
+				.click();
+
+			await maybeSetRecoveryAnswer(page);
+
+			await maybeAcceptTermsOfUse(page);
+
+			await expect(page.getByText('Welcome to Liferay')).toBeVisible();
+
+			await page.context().storageState({path: storageStatePath});
+
+			loggedIn = true;
+		}
+		else {
+			const {cookies} = JSON.parse(readTempFile('storageState.json'));
+
+			page.context().addCookies(cookies);
+		}
+
+		const cookies = await page.context().cookies();
+
+		await use({
+			password,
+			sessionId: cookies.find((cookie) => cookie.name === 'JSESSIONID')
+				.value,
+			user,
+		});
+	},
+});
+
+async function maybeAcceptTermsOfUse(page: Page) {
```

Comment:
Why `maybe` in the name of two tests? To me, this would imply some random chance in the test, which would be a rather strange test :slightly_smiling_face:

Reply:
We use that in other places -> https://github.com/search?q=repo%3Aliferay%2Fliferay-frontend-projects+maybe&type=code

Reply:
BTW, they are not tests but helper functions. Tests begin with `test(....`

Reply:
> We use that in other places

Still weird, at least to me. I typically don't want my code do _maybe_ do something, unless we deploy this on quantum computers :smile: 

Literally all functions have a chance to fail, with memory overflow if nothing else. It doesn't mean that all functions should have `maybe` in name.

I'd use `maybe` only if I intentionally code in chance to do something. Might be just me though :man_shrugging:

Reply:
I'm going to remove those functions in any case, because they are causing issues with the CI testing, so we don't need to care any more about them :partying_face: